components/HomeScreen/Title/Title.jsx

```diff
@@ -0,0 +1,44 @@
+import { Text, TouchableWithoutFeedback } from "react-native";
+import { theme } from "../../../utils/theme";
+
+
+/**
+ * Title.
+ *
+ * @param {object} props
+ * @param {number} props.globeOrientation
+ * @param {Function} props.onPress
+ */
+function Title(props) {
+
+  const {
+    globeOrientation,
+    onPress,
+  } = props;
+
+  const globe = globeOrientation === 1 ? "🌎" :
```

Comment:
In general don't nest ternaries every nest makes it significantly harder to parse at a glance, even though it's more verbose using if/else in this case is easier to follow. Might also consider extracting this to a small util